src/js/EpisodesSearch.tsx

```diff
@@ -54,8 +55,23 @@ const PaginationSpacer = styled.div`
 `;
 
 export const EpisodeSearch = () => {
-  const [searchTerm, setSearchTerm] = useState('');
-  const [page, setPage] = useState(0);
+  const [isMounted, setIsMounted] = useState(false);
+  const [searchParams, setSearchParams] = useSearchParams();
+  const [searchTerm, setSearchTerm] = useState(
+    searchParams.get('search') || ''
+  );
+  const [page, setPage] = useState(parseInt(searchParams.get('page') ?? '0'));
```

Comment:
I'm not convinced having page number in the URL is a good idea. The tricky thing is that:

1. It affects browser history so changing pages 10 times adds 10 new items to browser history and would mean when you pressed "back" in the browser, it would simply go back a page in the _search results_. This might be desirable but I'm not sure! This can also be disabled with a `history.replace` but...
2. Page numbers tend to stick around in the URL after you no longer want them. So for example you bookmark `transcript.fish` but you're not super tech savvy so you bookmark page 2 by accident. Or share page 2 by accident and that leads to a kind of confusing experience.
3. Sharing links to a specific page of a specific search seems of limited use. That search will potentially produce a different result in a week so you're basically guaranteed for that page to have different results than the ones you meant to link to.